### PR TITLE
chore(prismic): add express path checker link

### PR DESCRIPTION
**Pull request by paul-vd**

### CODE REVIEW

File: source/docs/prismic/routable-types.md [hunk 1]
```diff
@@ -126,8 +126,9 @@ If you have a nested route e.g. `/albums/:category/:uid` then you need add the `
 ```
 
 <blockquote class="warning">
-**Depth Limit** The Route Resolver is limited to retrieving data from 3 levels deep, please see the [Route Resolver example](https://prismic.io/docs/technologies/route-resolver-nuxtjs#route-resolver-examples) for more information.
+**Depth Limit:** The Route Resolver is limited to retrieving data from 2 levels deep, please see the [Route Resolver example](https://prismic.io/docs/technologies/route-resolver-nuxtjs#route-resolver-examples) for more information.
```
Comment (real34) [inline source/docs/prismic/routable-types.md:129]: **question:** why this change? The Prismic documentation still mentions 3 levels deep.

Comment (paul-vd) [reply source/docs/prismic/routable-types.md:129]: The Prismic documentation seems to be a little confusing and not well defined regarding the resolver, after exploring this in more detail we found the error reporting the following: 🤷
```
The resolver `level3` can only resolve a document link 2 level deep.
```



Here is an example that better explains the levels, and it might be good to include it in our docs?

```
// Level 1 (This is the actual document in question)
{
   type : "album"
   title: "System of a Down",
   uid: "soad",
   parent_category: "hard-rock", 
}

// Level 2 (Relationship to album)
{
    type: "category",
    title: "Hard Rock",
    uid: "hard-rock"
    parent: "rock"
}

// level 3 (Relationship to category)
{
   type: "category",
   title: "Rock",
   uid: "rock",
   parent: "music"
}
```

This example will fail with the error mentioned above:

```js
    loaders.Prismic.registerRoutableType({
      typeIdentifier: "album",
      urlFieldName: "uid",
      graphQLType: "Album",
      path: "/:level1?/:level2?/:level3?/:uid",
      resolvers: {
        level1: "parent_category",
        level2: "parent_category.parent",
        level3: "parent_category.parent.parent", ❌ this is too deep 🌊
      },
    });

    loaders.Prismic.registerRoutableType({
      typeIdentifier: "category",
      urlFieldName: "uid",
      graphQLType: "AlbumCategory",
      path: "/:parent?/:uid",
      resolvers: {
        parent: "parent",
      },
    });
```

Returns:
```
{
  type: "album",
  uid: "soad",
  url: "/rock/hard-rock/soad",
  parent_category:{
     type: category',
     uid: 'hard-rock',
     url: '/rock/hard-rock',
  }
}
```


One notable limitation is that we can not use dynamic nested routes that go beyond this depth,
```
path: /:category*/:uid
url: /music/rock/hard-rock/alternative/soad
```

If I recall correctly from a debug session, Prismic also does not support dynamic nested routes.

Comment (paul-vd) [reply source/docs/prismic/routable-types.md:129]: @real34 just checking if I can resolve this conversation and a review plz 🙏

Comment (mo-fc) [reply source/docs/prismic/routable-types.md:129]: > The resolver `level3` can only resolve a document link 2 level deep.

Very nice :D

Comment (real34) [reply source/docs/prismic/routable-types.md:129]: Sure! Sorry.

Thanks for the explanations.

**Review by real34 — APPROVED** [review]
